packages/react/src/components/Debug/Debug.tsx

```diff
@@ -0,0 +1,193 @@
+import keyboardKey from 'keyboard-key'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+import { toRefObject } from '@stardust-ui/react-component-ref'
+import { EventListener } from '@stardust-ui/react-component-event-listener'
+
+import { isBrowser } from '../../lib'
+
+import DebugPanel from './DebugPanel'
+import FiberNavigator from './FiberNavigator'
+import DebugRect from './DebugRect'
+
+type DebugProps = {
+  /** Existing document the popup should add listeners. */
+  mountDocument?: Document
+}
+
+type DebugState = {
+  debugPanelPosition?: 'left' | 'right'
+  fiberNav: FiberNavigator
+  selectedFiberNav: FiberNavigator
+  isSelecting: boolean
+}
+
+const INITIAL_STATE: DebugState = {
+  fiberNav: null,
+  selectedFiberNav: null,
+  isSelecting: false,
+}
+
+class Debug extends React.Component<DebugProps, DebugState> {
+  state = INITIAL_STATE
+
+  static defaultProps = {
+    // eslint-disable-next-line no-undef
+    mountDocument: isBrowser() ? window.document : null,
+  }
+
+  static propTypes = {
+    mountDocument: PropTypes.object.isRequired,
+  }
+
+  constructor(p, s) {
+    super(p, s)
+    if (isBrowser()) {
+      // eslint-disable-next-line
```

Comment:
```suggestion
      // eslint-disable-next-line no-undef
```

Let's disable rules explicitly